scripts/release/README.md

```diff
@@ -1,14 +1,16 @@
-# Helper for monitoring of release
-`pip install .`
-`monitor-release --help`
+# release-monitor
 
-Check current status of containers
-`monitor-release master containers`
-Check running sidecars:
-`monitor-release master sidecars`
 
-# Create .env file
-```
+Helper for monitoring of release. Here some one-liner examples:
+
+- Installation: `pip install .; monitor-release --help`
```

Comment:
You have a makefile now this is wrong right?

Reply:
updated